common/src/test/java/gov/cms/ab2d/common/service/CoverageServiceImplTest.java

```diff
@@ -99,6 +108,7 @@ public void insertContractAndDefaultCoveragePeriod() {
 
     @AfterEach
     public void cleanUp() {
+        coverageDeltaTestRepository.deleteAll();
```

Comment:
Does cleaning this up really need to be part of the DataSetup.cleanup method? There are lots of other integration tests that could potentially generate deltas.

Reply:
I toyed with several approaches.  Upon further reflection, I will move this to dataSetup.cleanup().